database_cleanup/model/purge_wizard.py

```diff
@@ -1,78 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
In order to save the log history of commits by files... Could you use `git mv` instead of `git rm && git add`?